dali/sasha/saxref.cpp

```diff
@@ -1161,16 +1199,14 @@ class CNewXRefManager: public CNewXRefManagerBase
 
     void clear()
     {
-        mem.reset();
-        delete root;
-        root = new cDirDesc(mem,"");
+        root = cDirDesc::create("");
```

Comment:
This is slightly dubious.  Assignment is not allowed, but I think it is an optimization that allows it to compile.